views/extras.tmpl

```diff
@@ -1,369 +1,341 @@
-                class="text-nowrap d-none d-md-inline-block pr-3"
-                href="{{.Links.Github}}"
-                title="dcrdata on GitHub"
-                target="_blank"
-                rel="noopener noreferrer"
-            >dcrdata v{{.Version}}</a>
-            <a
-                class="text-nowrap"
-                href="{{.Links.License}}"
-                target="_blank"
-                rel="noopener noreferrer"
-            >© 2019 The Decred developers (ISC)</a>
-        </span>
-        <span
-            id="connection"
-            class="text-nowrap align-items-center clickable d-inline-block"
-            data-turbolinks-permanent
-            data-controller="connection"
-            data-target="connection.indicator"
-            data-action="click->connection#requestNotifyPermission"
-            title="While connected, you will receive live page updates and, if enabled, desktop notifications (click to enable)."
-            ><span class="d-none d-sm-inline" data-target="connection.status">Connecting <span class="d-none d-md-inline-block">to WebSocket...</span></span><div></div>
-        </span>
-    </div>
-    <script
-        src="/dist/js/vendors~app.bundle.js"
-        data-turbolinks-eval="false"
-        data-turbolinks-suppress-warning
-    ></script>
-    <script
-        src="/dist/js/app.bundle.js"
-        data-turbolinks-eval="false"
-        data-turbolinks-suppress-warning
-    ></script>
+	<div class="container d-flex justify-content-between align-items-center">
+		<span>
+			<span data-controller="time" data-target="time.blocktime" data-stamp="{{$.Tip.Time}}"></span> <span class="d-none d-sm-inline">since last block</span>
+		</span>
+		<span>
+			<a
+				class="text-nowrap d-none d-md-inline-block pr-3"
+				href="{{.Links.Github}}"
+				title="dcrdata on GitHub"
+				target="_blank"
+				rel="noopener noreferrer"
+			>dcrdata v{{.Version}}</a>
+			<a
+				class="text-nowrap"
+				href="{{.Links.License}}"
+				target="_blank"
+				rel="noopener noreferrer"
+			>© 2019 The Decred developers (ISC)</a>
+		</span>
+		<span
+			id="connection"
+			class="text-nowrap align-items-center clickable d-inline-block"
+			data-turbolinks-permanent
+			data-controller="connection"
+			data-target="connection.indicator"
+			data-action="click->connection#requestNotifyPermission"
+			title="While connected, you will receive live page updates and, if enabled, desktop notifications (click to enable)."
+			><span class="d-none d-sm-inline" data-target="connection.status">Connecting <span class="d-none d-md-inline-block">to WebSocket...</span></span><div></div>
+		</span>
+	</div>
+	<script
+		src="/dist/js/vendors~app.bundle.js"
+		data-turbolinks-eval="false"
+		data-turbolinks-suppress-warning
+	></script>
+	<script
+		src="/dist/js/app.bundle.js"
+		data-turbolinks-eval="false"
+		data-turbolinks-suppress-warning
+	></script>
 </footer>
 {{end}}
 
-{{define "decimalParts"}}
+{{define "decimalParts" -}}
```

Comment:
`decimalParts`, `fmtPercentage`, and `hashElide` are "microtemplates" that get executed perhaps thousands of times, plus the indentation is very unlikely to match the surrounding html.  Thus, I think it's best to have these execute into html on a single line.